### ISSUE TITLE
Requirements

### ISSUE THREAD

**Issue by slester** [habnabit/passacre#3]
Should it be noted somewhere that you need libffi? I just tried installing this on OS X and it failed, looking for libffi. After I installed it with brew, everything worked as planned.

**Comment by habnabit** [comment]
That is a good point! It's often installed, but it's certainly useful to call these things out explicitly.

I'll definitely keep this in mind when working on docs again. If you don't mind, I'll just keep the issue open until then.

**Comment by slester** [comment]
@habnabit of course! I really love this project; let me know if there's anything I can do to help! I'll even write documentation. :)
